snap/snapcraft.yaml

```
@@ -22,27 +22,27 @@ apps:
       # https://docs.snapcraft.io/environment-variables/7983
       HOME: $SNAP_USER_COMMON
   qt:
-    command: desktop-launch bitcoin-qt
+    command: bin/desktop-launch bin/bitcoin-qt
     plugs: [home, removable-media, network, network-bind, desktop, x11, unity7]
```

```suggestion
    plugs: [home, removable-media, network, network-bind, desktop, x11, wayland, unity7]
```
Yes, it depends on first from a building application and deploy qt plugins, but x11 now is deprecated... Wayland, more relevant for linux desktops